hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestPayloadDeprecationFlow.scala

```diff
@@ -138,8 +163,8 @@ class TestPayloadDeprecationFlow extends SparkClientFunctionalTestHarness {
 
     // 5. Add a delete.
     val fourthUpdateData = Seq(
-      (12, 3L, "rider-CC", "driver-CC", 33.90, "i", "12.1", 12, 1),
-      (12, 5L, "rider-EE", "driver-EE", 17.85, "i", "12.1", 12, 1))
+      (12, 3L, "rider-CC", "driver-CC", 33.90, "i", "12.1", 12, 1, "i"),
+      (12, 5L, "rider-EE", "driver-EE", 17.85, "i", "12.1", 12, 1, "i"))
```

Comment:
Should these be `d` and let the operation be `upsert` instead of `delete`, or here the deletes are intended to be always commit time ordered (which `delete` operation follows)?

Reply:
I didn't write these tests so I don't know. I'm just pattern matching the `Op` field here

Reply:
OK, we can take this as a follow-up @linliu-code do you know the original intention?